examples/httpd_example/priv/index.html

```diff
@@ -0,0 +1,191 @@
+"THE BEER-WARE LICENSE" (Revision 42):
+<fred@dushin.net> wrote this file.  You are hereby granted permission to
+copy, modify, or mutilate this file without restriction.  If you create a
+work derived from this file, you may optionally include a copy of this notice,
+for which I would be most grateful, but you are not required to do so.
+If we meet some day, and you think this stuff is worth it, you can buy me a
+beer in return.   Fred Dushin
+----------------------------------------------------------------------------
+-->
+<!DOCTYPE html>
+<html>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<!-- <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css"> -->
+	<link rel="stylesheet" href="w3.css">
+	<head>
+		<title>AtomVM Web Console</title>
+		<style>
+			#loader {
+				position: absolute;
+				left: 50%;
+				top: 50%;
+				z-index: 1;
+				width: 100px;
+				height: 100px;
+				margin: -75px 0 0 -75px;
+				border: 16px solid #f3f3f3;
+				border-radius: 50%;
+				border-top: 16px solid #3498db;
+				width: 120px;
+				height: 120px;
+				-webkit-animation: spin 2s ease infinite;
+				animation: spin 2s ease infinite;
+			}
+			@-webkit-keyframes spin {
+				0% { -webkit-transform: rotate(0deg); }
+				100% { -webkit-transform: rotate(360deg); }
+			}
+			@keyframes spin {
+				0% { transform: rotate(0deg); }
+				100% { transform: rotate(360deg); }
+			}
+			.animate-bottom {
+				position: relative;
+				-webkit-animation-name: animatebottom;
+				-webkit-animation-duration: 1s;
+				animation-name: animatebottom;
+				animation-duration: 1s
+			}
+			@-webkit-keyframes animatebottom {
+				from { bottom:-100px; opacity:0 }
+				to { bottom:0px; opacity:1 }
+			}
+			@keyframes animatebottom {
+				from{ bottom:-100px; opacity:0 }
+				to{ bottom:0; opacity:1 }
+			}
+			#content {
+				display: none;
+			}
+			#icon {
+				text-align: right;
+				padding: 5;
+  				margin-top: 5px;
+  				margin-right: 5px;
+  			}
+		</style>
+	</head>
+	<body onload="timedShowContents()">
+		<div id="loader"></div>
+		<div id="content" class="w3-panel animate-bottom" style="display:none;">
+			<div id="tabs" class="w3-container" style="min-height:600px">
+
+				<div class="w3-bar w3-blue-gray">
+					<button class="w3-bar-item w3-button" onclick="openPanel(event, 'about')">About</button>
+					<button class="w3-bar-item w3-button" onclick="openPanel(event, 'system')">System</button>
+					<button class="w3-bar-item w3-button" onclick="openPanel(event, 'memory')">Memory</button>
+					<div id="icon"><img src="favicon.ico" alt="AtomVM logo" width="30" height="30" /></div>
+				</div>
+
+				<div id="about" class="w3-panel w3-animate-opacity wc-panel">
+					<div id="about-view">
+						Welcome to the AtomVM web console.
+
+						Use this console to explore the current state of your device.
+						<!-- <img src="AVM-logo.png" alt="AtomVM logo" width="300" height="300"/> -->
```

Comment:
Was this something that should be removed? ...or uncommented?

Reply:
Removed all that commented out code.  Thanks for spotting.